### ISSUE TITLE
the ovs-tcpdump cli can't be use in ovs pod 

### ISSUE THREAD

**Issue by lynn901** [kubeovn/kube-ovn#2397]
# Expected Behavior
ovs-tcpdump run normally
# Actual Behavior
ovs-tcpdump run unnormally
# Steps to Reproduce the Problem

1. kubectl exec -it ovs-ovn-xxx  -nkube-system -- bash 
1. ovs-tcpdump

# Additional Info

- Kubernetes version:

  **Output of `kubectl version`:**

  ```
  (paste your output here)
  ```

- kube-ovn version:
  
  ```
  (paste your output here)
  ```

- operation-system/kernel version:
  
  **Output of `awk -F '=' '/PRETTY_NAME/ { print $2 }' /etc/os-release`:**
  **Output of `uname -r`:**
  ```
  (paste your output here)
  ```

**Comment by oilbeater** [comment]
The `python3-openvswitch` has already been build and installed in the base image and the `ovs-tcpdump` works well in my env. What's the version of image that you found the issue?

**Comment by lynn901** [comment]
> The `python3-openvswitch` has already been build and installed in the base image and the `ovs-tcpdump` works well in my env. What's the version of image that you found the issue?

![image](https://user-images.githubusercontent.com/11764870/222023901-531c8b80-e1e5-4906-b20e-6be84079c12e.png)
v.1.11.1

**Comment by oilbeater** [comment]
The `python3-openvswitch` will be installed in `/usr/lib/python3.7` and the base has been updated to `python3.10` we should move the `ovs` dir to `/usr/lib/python3.10`. And the master doesn't has this problem as the ovs have been upgraded and resolved this issue.

The related fix should be pr to only release-1.11

**Comment by lynn901** [comment]
I also saw this problem in other versions, eg. v1.10.6 and v1.10.4
